From fa848c31ba3e74d05b9408506f553d5612031de8 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Mon, 12 Oct 2015 02:04:11 -0700
Subject: [PATCH 6/8] XQuartz: Relax App Transport Security for communicating
 with the update server

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit df80e2649aa47957986554928e6bf41b86c15240)
---
 hw/xquartz/bundle/Info.plist.cpp | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/hw/xquartz/bundle/Info.plist.cpp b/hw/xquartz/bundle/Info.plist.cpp
index a0d9050..06e33f8 100644
--- a/hw/xquartz/bundle/Info.plist.cpp
+++ b/hw/xquartz/bundle/Info.plist.cpp
@@ -35,6 +35,19 @@
 		<string>sparkle.pem</string>
         <key>SUFeedURL</key>
                 <string>http://xquartz.macosforge.org/downloads/sparkle/release.xml</string>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>macosforge.org</key>
+			<dict>
+				<key>NSIncludesSubdomains</key>
+				<true/>
+				<key>NSExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+			</dict>
+		</dict>
+	</dict>
 #endif
 	<key>LSApplicationCategoryType</key>
 		<string>public.app-category.utilities</string>
-- 
2.6.1

